src/MobileUI/Features/Settings/DeleteProfilePage.xaml

```diff
@@ -95,7 +95,7 @@
                                Grid.Column="0"
                                Grid.ColumnSpan="2"
                                Grid.RowSpan="3"
-                               IsRunning="True"
```

Comment:
I noticed this a few times, any reasons why we went from `True` to `False`?

Reply:
Ok, I think it's because IsRunning and IsVisible seem to have same value and IsVisible doesn't start off the same state as IsVisible from the get go. As far as I understand, previously, it was constantly running, potentially causing perf issues even if user didn't see the running animation.

Good change, although, if they are interlinked, maybe we should use data binding on a single variable instead. (not now, just an idea for future)